javascript/javascript_sdk/src/sensr_client.js

```diff
@@ -0,0 +1,56 @@
+const WebSocket = require('ws');
+const outputMsg = require('./../js_proto/sensr_proto/output_pb.js');
+const pointMsg = require('./../js_proto/sensr_proto/point_cloud_pb');
+
+const SensrClient = endpoint => {
```

Comment:
Just curious, is there a reason you preferred a function over a class for this?

Reply:
No technical reasons, I just felt more comfortable using function